R/data-table.R

```diff
@@ -15,11 +15,15 @@ use_data_table <- function() {
   check_is_package("use_data_table()")
   check_uses_roxygen("use_data_table()")
 
-  deps <- desc::desc_get_deps(proj_get())
+  desc <- proj_desc()
+  deps <- desc$get_deps()
+
   if (any(deps$type == "Depends" & deps$package == "data.table")) {
     ui_warn("data.table should be in Imports or Suggests, not Depends")
     ui_done("Deleting data.table from {ui_field('Depends')}")
-    desc::desc_del_dep("data.table", "Depends", file = proj_get())
+
+    desc$del_dep("data.table", "Depends")
+    desc$write()
```

Comment:
Once this is done I'll tackle #1379 by making `$write()` automatically tidy the description if some flag is set.